pkg/prom/cluster/node.go

```diff
@@ -137,37 +140,6 @@ func (n *node) ApplyConfig(cfg Config) error {
 	return nil
 }
 
-// WaitJoined waits for the node the join the cluster and enter the
```

Comment:
I moved this function down towards the end of the file so everything here was in usage-order.